app/app.go

```diff
@@ -63,6 +62,66 @@ func New(options ...AppOption) *App {
 	return app
 }
 
+func (a *App) InitServer() {
+	a.srv.AppInitialized.Do(func() {
```

Comment:
This is one of the key changes. I running this the first time I instantiate the App, so we initialize the server part that is strictly dependent on the App itself.